hazelcast-client-new/src/test/java/com/hazelcast/client/ClientConnectionTest.java

```diff
@@ -42,36 +41,37 @@
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(HazelcastSerialClassRunner.class)
+@RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientConnectionTest extends HazelcastTestSupport {
 
+    private final TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
```

Comment:
Can you make a ticket for it? Then I'm happy to merge it. I'm even willing to help out changing the client tests so they can make use of the server side tests. This way you get a lot of duplicate test code removed